src/log-store/src/kafka/log_store.rs

```diff
@@ -170,39 +173,64 @@ impl LogStore for KafkaLogStore {
             ns, start_offset, end_offset
         );
 
+        // Key: entry id, Value: the records associated with the entry.
+        let mut entry_records: HashMap<_, Vec<_>> = HashMap::new();
         let ns_clone = ns.clone();
         let stream = async_stream::stream!({
             while let Some(consume_result) = stream_consumer.next().await {
-                // Each next will prdoce a `RecordAndOffset` and a high watermark offset.
+                // Each next on the stream consumer produces a `RecordAndOffset` and a high watermark offset.
                 // The `RecordAndOffset` contains the record data and its start offset.
                 // The high watermark offset is the end offset of the latest record in the partition.
```

Comment:
I double-check this `high watermark offset`, it seems should be the `last offset + 1`(and same as the `get_offset`)

Reply:
It's not used actually. Only for logging.

Reply:
Will update the comment on the high watermark offset.